examples/opencv/goclient-example/opencv-example.go

```diff
@@ -0,0 +1,116 @@
+package main
+
+// This is an OpenCV example written in Go. Run this file
+// from the root of the repo. You must have a working
+// Pachyderm cluster running on your machine to run this
+// example.
+
+import (
+	"fmt"
+	"github.com/pachyderm/pachyderm/src/client"
+	"github.com/pachyderm/pachyderm/src/client/pfs"
+	"github.com/pachyderm/pachyderm/src/client/pps"
+	"os"
+)
+
+func main() {
+
+	// Replace the IP address with your `pachd` address.
+	// If running in minikube, this will be your minikube
+	// IP.
+	c, err := client.NewFromAddress("192.168.64.2:30650")
+	if err != nil {
+		panic(err)
+	}
+
+	if _, err := c.PfsAPIClient.CreateRepo(
+		c.Ctx(),
+		&pfs.CreateRepoRequest{
+			Repo:        client.NewRepo("images"),
+			Description: "An images repo",
+			Update:      true,
+		},
+	); err != nil {
+		panic(err)
+	}
+
+	file1, err := os.Open("examples/opencv/liberty.png")
+	if err != nil {
+		panic(err)
+	}
+
+	if _, err := c.PutFile("images", "master", "liberty.png", file1); err != nil {
+		panic(err)
+	}
+
+	file2, err := os.Open("examples/opencv/AT-AT.png")
+	if err != nil {
+		panic(err)
+	}
+
+	if _, err := c.PutFile("images", "master", "AT-AT.png", file2); err != nil {
+		panic(err)
+	}
+
+	file3, err := os.Open("examples/opencv/kitten.png")
+	if err != nil {
+		panic(err)
+	}
+
+	if _, err := c.PutFile("images", "master", "kitten.png", file3); err != nil {
```

Comment:
I don't think  it's productive to include the images in the repo.  Should you use a call to PutFileURL, instead?